contracts/ERC721BridgelessMinting.sol

```diff
@@ -81,7 +82,7 @@ contract ERC721BridgelessMinting is ERC721 {
 
     /**
      * @notice Returns the current owner of a token
-     * @dev The function overrides the one in the base ERC721 contract, and it returns the token's correct owner if the token has never been previously transferred.
+     * @dev @dev This function overrides the one in the base ERC721 contract. On deploy, all tokens have an assigned owner, encoded in their tokenId, and determined via usage of initOwner(tokenId). Upon transfer, the new owner is stored in, and retrieved from, the contract storage.
```

Comment:
is this the way very long comment lines work? Don't you need to split it into multiple lines following the solidity convention? check: https://www.bitdegree.org/learn/solidity-syntax